### PR TITLE
adds comment count to finding articles by id

### PR DESCRIPTION
**Pull request by PGallagher93**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ describe("GET:/api/articles/:article_id", () => {
         expect(article).toHaveProperty("created_at");
         expect(article).toHaveProperty("votes");
         expect(article).toHaveProperty("article_img_url");
+        expect(article).toHaveProperty("comment_count")
```
Comment (johnhutcheon) [inline __tests__/app.test.js:61]: We shouldn't be editing an earlier test to pass this later ticket. Could you create a new test please? 🙂

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hey Phil, could you just see my comment and amend the testing so that it's a new test please? Cheers!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -142,11 +143,19 @@ describe("GET 200: /api/articles", () =>{
...
     })
   })
+  test("The returning article objects have a key of comment_count", () =>{
+    return request(app).get("/api/articles").expect(200).then(({body}) =>{
+      const {articles} = body
+      
+      articles.forEach((article) => {
+        expect(article).toHaveProperty("comment_count")
+      })
+  })})
```
Comment (Harrison365) [inline __tests__/app.test.js:158]: Nice 👍

**Comment by Harrison365** [comment]
Good change ✅ 
Merge when ready 🎉

**Pull request merged**